### ISSUE TITLE
How to set the learningrate when training CUHK-PEDES?

### ISSUE THREAD

**Issue by jingyaa** [layumi/Image-Text-Embedding#4]
Hi,
I want to train the CUHK-PEDES using your method, in your paper "In the first training stage, we fixed the pre-trained image CNN, and train the text CNN only. The learning rate is 0.001.", [https://github.com/layumi/Image-Text-Embedding/blob/master/train_cuhk_word2_1_pool.m#L34](url)
and
[https://github.com/layumi/Image-Text-Embedding/blob/master/cuhk_word2_pool.m#L269](url)
the learning rate is 0.1 and 0.001, how should I set the learning rate?

**Comment by layumi** [comment]
Hi @jingyaa 
Sorry for the late response.
In matconvnet, the learning rate = global learning rate* layer learning rate.
You may find the default layer learning rate at https://github.com/layumi/Image-Text-Embedding/blob/master/matlab/%2Bdagnn/%40DagNN/initParams.m

For text CNN, the learning rate of the final conv layer is 0.1*0.01 = 0.001, 
the other conv layer 0.1*0.1=0.01. For simplify, I just wrote the final learning rate 0.001
I will check again and make it clear in the revised version. Thank you!
